atm.js

```diff
@@ -1,21 +1,99 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
 
-var checkingDeposit = function(amount){
-  checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
 
-}
+// ======= init ======= 
+var Cbalance = parseInt($('.balance').eq(0).html().substring(1));
+var Sbalance = parseInt($('.balance').eq(1).html().substring(1));
 
-function checkingWithdrawal(amount){
 
-}
+// ======= buttons ======= 
+var Cdeposit = $('.account input').eq(1).on('click', function(event) {
+	make_transaction('checking', 'deposit');
+});
 
-function savingsDeposit(amount){
+var Sdeposit = $('.account input').eq(4).on('click', function(event) {
+	make_transaction('savings', 'deposit')
+});
 
-}
+var Cwithdraw = $('.account input').eq(2).on('click', function(event){
+	make_transaction('checking', 'withdraw');
+});
+
+var Swithdraw = $('.account input').eq(5).on('click', function(event){
+	make_transaction('savings', 'withdraw');
+});
+
+
+// ======= make_transaction =======	
+function make_transaction(whichAccount, dep_Or_wdrw) {
+
+	// ======= input fields ======= 
+	var CdepositAmount = $('.account input').eq(0).val();
+	var SdepositAmount = $('.account input').eq(3).val();
+
+	// ======= validation (text entry) ======= 
+	if ((isNaN(CdepositAmount) && CdepositAmount != '') || (isNaN(SdepositAmount) && SdepositAmount != '')) {
+		$('.balance').eq(0).addClass("message");
+		$('.balance').eq(0).html('numbers only please');
+	} else {
```

Comment:
Anytime I have some code that does a single job, I like to break it out into a separate function, i.e. here I might move this code into a function `validateNumerical` and then call that function here instead. Makes the code more readable, maintainable, and flexible.